x-pack/plugins/alerting/server/task_runner/task_runner.test.ts

```diff
@@ -439,259 +263,74 @@ describe('Task Runner', () => {
-          "executionId": "5f6aa57d-3e22-484e-bae8-cbed868f4d28",
-          "id": "1",
-          "params": Object {
-            "foo": true,
-          },
-          "relatedSavedObjects": Array [
-            Object {
-              "id": "1",
-              "namespace": undefined,
-              "type": "alert",
-              "typeId": "test",
-            },
-          ],
-          "source": Object {
-            "source": Object {
-              "id": "1",
-              "type": "alert",
-            },
-            "type": "SAVED_OBJECT",
-          },
-          "spaceId": undefined,
-        },
-      ]
-    `);
+      expect(enqueueFunction).toHaveBeenCalledWith(generateEnqueueFunctionInput());
 
       const logger = customTaskRunnerFactoryInitializerParams.logger;
       expect(logger.debug).toHaveBeenCalledTimes(4);
       expect(logger.debug).nthCalledWith(1, 'executing rule test:1 at 1970-01-01T00:00:00.000Z');
       expect(logger.debug).nthCalledWith(
         2,
-        `rule test:1: 'rule-name' has 1 active alerts: [{\"instanceId\":\"1\",\"actionGroup\":\"default\"}]`
+        `rule test:1: '${RULE_NAME}' has 1 active alerts: [{\"instanceId\":\"1\",\"actionGroup\":\"default\"}]`
       );
       expect(logger.debug).nthCalledWith(
         3,
         'ruleExecutionStatus for test:1: {"numberOfTriggeredActions":1,"lastExecutionDate":"1970-01-01T00:00:00.000Z","status":"active"}'
       );
-      // ruleExecutionStatus for test:1: {\"lastExecutionDate\":\"1970-01-01T00:00:00.000Z\",\"status\":\"error\",\"error\":{\"reason\":\"unknown\",\"message\":\"Cannot read property 'catch' of undefined\"}}
 
       const eventLogger = customTaskRunnerFactoryInitializerParams.eventLogger;
       expect(eventLogger.logEvent).toHaveBeenCalledTimes(5);
-      expect(eventLogger.logEvent).toHaveBeenNthCalledWith(1, {
-        event: {
-          action: 'execute-start',
-          category: ['alerts'],
-          kind: 'alert',
-        },
-        kibana: {
-          alert: {
-            rule: {
-              execution: {
-                uuid: '5f6aa57d-3e22-484e-bae8-cbed868f4d28',
-              },
-            },
-          },
-          task: {
-            schedule_delay: 0,
-            scheduled: '1970-01-01T00:00:00.000Z',
-          },
-          saved_objects: [
-            {
-              id: '1',
-              namespace: undefined,
-              rel: 'primary',
-              type: 'alert',
-              type_id: 'test',
-            },
-          ],
-        },
-        message: `rule execution start: "1"`,
-        rule: {
-          category: 'test',
-          id: '1',
-          license: 'basic',
-          ruleset: 'alerts',
-        },
-      });
-      expect(eventLogger.logEvent).toHaveBeenNthCalledWith(2, {
-        event: {
-          action: 'new-instance',
-          category: ['alerts'],
-          kind: 'alert',
+      expect(eventLogger.logEvent).toHaveBeenNthCalledWith(
```

Comment:
This might be an unpopular opinion, especially because this PR does remove so many LOC, which is amazing! But I find it helpful to see the full event log entry in these tests when I'm adding a feature or fixing a bug. Especially since all event log fields are optional, so we can't depend on the typings to tell us if there's something missing or we're accidentally missing a field in the event log. If no one else feels this way, I am happy to defer though.

Reply:
@elastic/response-ops-execution Any thoughts on the above comment?

Reply:
I am not opinionated about these type of things, if the team is ok i am ok too :)

Reply:
I'm willing to try it out :)

Reply:
ya, happy to try this out, we can go back to the old way later if needed ...